lib/libzutil/zutil_pool.c

```diff
@@ -120,8 +121,9 @@ zpool_history_unpack(char *buf, uint64_t bytes_read, uint64_t *leftover,
 			break;
 
 		/* unpack record */
-		if (nvlist_unpack(buf + sizeof (reclen), reclen, &nv, 0) != 0)
-			return (ENOMEM);
+		err = nvlist_unpack(buf + sizeof (reclen), reclen, &nv, 0);
```

Comment:
1. I'd reduce the scope of `err`
```suggestion
		int err = nvlist_unpack(buf + sizeof (reclen), reclen, &nv, 0);
```
2. `error: commit subject over 72 characters`